lib/utils.js

```diff
@@ -219,16 +219,67 @@ utils.findRefs = (json, absolutePath, relativePath, initialFormat, asyncapiYAMLo
 utils.getLocationOf = (jsonPointer, asyncapiYAMLorJSON, initialFormat) => {
   const ast = getAST(asyncapiYAMLorJSON, initialFormat);
   if (!ast) return { jsonPointer };
+
   return findLocationOf(jsonPointerToArray(jsonPointer), ast, initialFormat);
 };
 
 utils.improveAjvErrors = (errors, asyncapiYAMLorJSON, initialFormat) => {
   const ast = getAST(asyncapiYAMLorJSON, initialFormat);
   return errors.map(error => {
     const defaultLocation = { jsonPointer: error.dataPath || '/' };
+    console.log(error.dataPath);
     return {
       title: `${error.dataPath || '/'} ${error.message}`,
       location: ast ? findLocationOf(jsonPointerToArray(error.dataPath), ast, initialFormat) : defaultLocation,
     };
   });
-};
\ No newline at end of file
+};
+
+/**
+ * Returns an array with all strings in between curly braces, including braces
+ */
+utils.getCurlyBracesEl = str => {
```

Comment:
Can we come up with a better name for this function? Outside the context of this PR, I'd not really understand what's doing.